src/pages/Signup/Signup.jsx

```diff
@@ -0,0 +1,29 @@
+import "./signup.css"
+
+export function Signup() {
+    return (
+        <div class="login-container">
+            <div class="login-container-wrapper">
+                <h2 class="login-page">Signup</h2>
+                <form>
+                    <div class="form-info">
+                        <label class="login-text" for="email">Email address:</label>
```

Comment:
Suggestion: You can use Name fields for signup

Reply:
Ok, I'll do that.